src/main/java/NullabilityRefactoring.java

```diff
@@ -0,0 +1,372 @@
+import org.eclipse.jdt.core.dom.*;
+import org.eclipse.jdt.core.dom.rewrite.ASTRewrite;
+import java.util.*;
+
+class NullabilityRefactoring extends Refactoring {
+
+    private Set<String> possiblyNullFields = new HashSet<>();
```

Comment:
What's the format of the `String`s in this map? Is it fully-qualified names or something else? (This should be addressed with a Javadoc comment, an annotation from the [Signature Checker](https://checkerframework.org/manual/#signature-checker) (even if you're not going to run the checker, you can use the annotations as documentation), or both.